.cirrus.yml

```diff
@@ -18,6 +18,16 @@ docker_builder:
   name: windows
   platform: windows
   os_version: 2019
+  matrix:
+    - name: Windows 1.4 tests
+      env:
+        ctrdVersion: 1.4.12
```

Comment:
I think we can drop v1.4 support for Windows

Reply:
sounds good, was thinking the same thing.  With out looking to deeply I think it might be related to https://github.com/containerd/containerd/commit/791e175c79930a34cfbb2048fbcaa8493fd2c86b